src/Chunk.ts

```diff
@@ -136,27 +136,27 @@ export default class Chunk {
 	suggestedVariableName: string;
 	variableName = '';
 
-	private accessedGlobalsByScope = new Map<ChildScope, Set<string>>();
+	private readonly accessedGlobalsByScope = new Map<ChildScope, Set<string>>();
 	private dependencies = new Set<ExternalModule | Chunk>();
-	private dynamicDependencies = new Set<ExternalModule | Chunk>();
-	private dynamicEntryModules: Module[] = [];
+	private readonly dynamicDependencies = new Set<ExternalModule | Chunk>();
+	private readonly dynamicEntryModules: Module[] = [];
 	private dynamicName: string | null = null;
-	private exportNamesByVariable = new Map<Variable, string[]>();
-	private exports = new Set<Variable>();
-	private exportsByName: Record<string, Variable> = Object.create(null);
+	private readonly exportNamesByVariable = new Map<Variable, string[]>();
+	private readonly exports = new Set<Variable>();
+	private readonly exportsByName: Record<string, Variable> = Object.create(null);
 	private fileName: string | null = null;
 	private implicitEntryModules: Module[] = [];
-	private implicitlyLoadedBefore = new Set<Chunk>();
-	private imports = new Set<Variable>();
+	private readonly implicitlyLoadedBefore = new Set<Chunk>();
+	private readonly imports = new Set<Variable>();
 	private indentString: string = undefined as never;
-	private readonly isEmpty: boolean = true;
```

Comment:
To my understanding it is ok to mark something as readonly if it is only changed in the constructor. It will prevent mutations in all other methods.